src/test/java/alinaTinca/tests/Tema7/MakeAppoitmentScenariosTests.java

```diff
@@ -0,0 +1,90 @@
+
+import java.util.List;
+
+public class MakeAppoitmentScenariosTests extends BaseTest{
+
+//Ex4
+    @DataProvider(name = "makeAppointment")
+    public Object[][] makeAppointmentDetails(){
+        return new Object[][] {
+            {"10/01/2023", "new comment", true},
+                //test case with all data filled - test passed!
+                {"22/02/2023", "", true},
+                //test case without the Comment field completed - test passed!
+                {"", "comm", false}
+                //test case without required field (Visit Date) filled - test failed!
+        };
+    }
+    @Test(dataProvider = "makeAppointment", description = "check different scenarios to make an appointment", groups = "app")
+    public void makeAppointment(String data, String comment, boolean testShouldPass){
+        WebElement menuButton = driver.findElement(By.cssSelector(".btn-dark.btn-lg.toggle"));
+        //find the Home Menu button
+        menuButton.click();
+        //click on home button
+        WebElement loginLink = driver.findElement(By.linkText("Login"));  //textul linkului numit Login
+        //find the Login button
+        loginLink.click();
+        //click on Login button
+        Assert.assertEquals(driver.getCurrentUrl(),
+                "https://katalon-demo-cura.herokuapp.com/profile.php#login",
+                //display login page after the login button has been opened
+                "Couldn't navigate to login page!");
+        //check if the user is redirected to login page when logged in and display an error message if the login button has been opened
+        WebElement usernameInput = driver.findElement(By.name("username"));
+        //find the username filed
+        usernameInput.sendKeys("John Doe");
+        //fill in the username field with the above username
+        WebElement passwordInput = driver.findElement(By.name("password"));
+        //find the password field
+        passwordInput.sendKeys("ThisIsNotAPassword");
+        //fill in password field with the above password
+        WebElement loginButton = driver.findElement(By.cssSelector("button[type='submit']"));
+        //find the login button
+        loginButton.click();
+        //click on login button
+        Select multipleSelectValueFacility = new Select(driver.findElement(By.xpath("/html/body/section/div/div/form/div[1]/div/select")));
+        //find the Facility Field
+        multipleSelectValueFacility.selectByValue("Tokyo CURA Healthcare Center");
+        //select the first item
+        WebElement applyForHospitalButton = driver.findElement(By.className("checkbox-inline"));
+        //find the hospital readmission checkbox
+        applyForHospitalButton.click();
+        //click on it
+        List<WebElement> healthcareProgramButton = driver.findElements(By.cssSelector("label[for='chk_hospotal_readmission']"));
+        //find the Healthcare Program radiobuttons
+        healthcareProgramButton.get(1);
```

Comment:
You can delete this line after changing from List to WebElement above

Reply:
I fixed 61 line